src/richie/apps/courses/cms_plugins.py

```diff
@@ -27,9 +28,10 @@ class OrganizationPlugin(CMSPluginBase):
     """
 
     cache = True
+    fieldsets = ((None, {"fields": ["page", "template"]}),)
     model = OrganizationPluginModel
     module = PLUGINS_GROUP
-    render_template = "courses/plugins/organization.html"
+    render_template = ORGANIZATION_TEMPLATES[0][0]
```

Comment:
maybe this is not necessary since you are overriding it below?

Reply:
It's to force a default and make sure that previously added plugins still work.

Reply:
But you're adding a field to the model with a default... and anyway you always override it so this should be useless.

Reply:
It's required for the build. Copied a comment from section_plugin indicating it.